server/server.go

```diff
@@ -53,15 +53,26 @@ func (s *Server) Status(instID flux.InstanceID) (res flux.Status, err error) {
 	}
 
 	res.Fluxsvc = flux.FluxsvcStatus{Version: s.version}
+
 	res.Fluxd.Version, err = inst.Platform.Version()
-	res.Fluxd.Connected = (err == nil)
+	if err != nil {
+		return res, err
+	}
+
+	res.Git.Config, err = inst.Platform.GitRepoConfig(false)
+	if err != nil {
+		return res, err
+	}
+
 	_, err = inst.Platform.SyncStatus("HEAD")
```

Comment:
Does the ff still need to be done to get this information -- won't the GitRepoConfig indicate whether it can successfully read the repo?